src/oscar/static_src/oscar/scss/page/product_lists.scss

```diff
@@ -0,0 +1,83 @@
+// No margin on lists
```

Comment:
Think we should be using the new [`no-gutters` class](https://getbootstrap.com/docs/4.0/layout/grid/#no-gutters) on such lists, instead of styling them manually here.